lib/cli/commands/show.js

```diff
@@ -7,7 +35,13 @@ module.exports = function (opts) {
       return yargs
     },
     handler: function (argv) {
-      argv.log.info('show')
+      const Arborist = require('@npmcli/arborist')
```

Comment:
I personally would prefer we not `require` deeply in code. While there might be some benefits sometimes, IMO they are all outweighed by the unexpected behavior of having a require fail only when running the program in a specific way.  If we hit a performance bottleneck where this require statement meaningfully slows down things, then worth considering IMO, but not until we do hit that.

Reply:
agreed will fix